docs/source/developer_guide/index.rst

```diff
@@ -4,80 +4,154 @@
 Contributor Guidelines
 ======================
 
-Getting Started
-###############
-
-Git and Github
-^^^^^^^^^^^^^^
+Thank you for your interest in contributing to PySyft! This document
+contains rules and a step-by-step approach to contributing to any PySyft
+repository.
 
-All our development is done using Git and Github. If you're not too familiar with Git and Github, `start by reviewing this guide <https://guides.github.com/activities/hello-world>`_.
+Getting Started
+***************
+
+The PySyft community admires and welcomes your expertise and enthusiasm!
+We are always excited to work with new contributors and engage with them
+to build exciting new features or improve existing bugs.
+
+If you are unsure where to start or how your skills fit in, take a look
+at open ``good-first-issues`` or introduce yourself in the #introductions
```

Comment:
I think we can convert this `good-first-issues` to a link to the currently open Good First Issues on our repo. What do you say?

Reply:
Yes, I thought the same, but currently, we have zero good first issues. We can definitely add this later when we have a list of good issues

Reply:
It's a good point, but I would say to add it now, this might fluctuate all the time anyway.